source/includes/stackpath/_cdn_settings.md

```diff
@@ -78,4 +78,95 @@ Attributes | &nbsp;
 `canonicalHeaderEnabled`<br/>*Boolean* | Whether or not to enable setting Link: <http://{hostname}/URI>; rel="canonical" header on each response.
 `canonicalHeader`<br/>*String* | The hostname for the canonicalHeader, only visible if `canonicalHeaderEnabled` is enabled. 
 `urlCachingEnabled`<br/>*Boolean* | Whether or not to enable caching of URLs without file extensions.
-`urlCachingTtl`<br/>*Integer* | The time to live for the url cache. Only visible if `urlCachingEnabled` is enabled. 
\ No newline at end of file
+`urlCachingTtl`<br/>*Integer* | The time to live for the url cache. Only visible if `urlCachingEnabled` is enabled. 
+
+
+<!-------------------- PURGE ALL CDN CACHED CONTENT -------------------->
+
+### Purge all CDN cached content
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/cdnsettings/9f236f19-55db-411f-9f05-bd79dc91a69b?operation=purgeAll"
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "taskId": "e6c03893-212f-4e28-b7a1-66ed7b554839",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/cdnsettings/:siteId?operation=purgeAll</a></code>
+
+Purge all CDN cached content of a site in a given [environment](#administration-environments).
+
+Attributes | &nbsp;
+------- | -----------
+`taskId` <br/>*string* | The task id related to the purge operation.
+`taskStatus` <br/>*string* | The status of the operation.
+
+<!-------------------- PURGE CUSTOM CDN CACHED CONTENT -------------------->
+
+### Purge custom CDN cached content
+
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/api/v1/services/stackpath/test-area/cdnsettings/9f236f19-55db-411f-9f05-bd79dc91a69b?operation=purge"
+```
+
+> Request body example:
+
+```json
+{
+	"items": [
+		{
+			"url": "//<domain_name>/path/to/file",
+      "recursive": true,
```

Comment:
Should fix the formatting on this one.

Reply:
55d53a151619bdaee2b2ea90d27cd789ea3bd177